cmd/dump.go

```diff
@@ -17,6 +17,7 @@ import (
 	"code.gitea.io/gitea/models"
 	"code.gitea.io/gitea/modules/setting"
 	"github.com/Unknwon/cae/zip"
+	"github.com/Unknwon/com"
```

Comment:
is there no simple standard way to check if a directory exists ? I'd try to keep number of dependencies down...